streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentInfoTest.java

```diff
@@ -30,7 +31,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-public class AssginmentInfoTest {
+public class AssignmentInfoTest {
```

Comment:
Could we add a unit test for compatibility: i.e. an old versioned serialized bytes can be correctly deser as well? Same for `SubscriptionInfo`.